test/e2e/page-objects/pages/hardware-wallet/connect-hardware-wallet-page.ts

```diff
@@ -0,0 +1,50 @@
+import { Driver } from '../../../webdriver/driver';
+
+class ConnectHardwareWalletPage {
+  private driver: Driver;
+
+  private readonly connectHardwareWalletPageTitle = {
+    text: 'Connect a hardware wallet',
+    tag: 'h3',
+  };
+
+  private readonly connectLatticeButton = '[data-testid="connect-lattice-btn"]';
+
+  private readonly connectTrezorButton = '[data-testid="connect-trezor-btn"]';
+
+  private readonly continueButton = { text: 'Continue', tag: 'button' };
+
+  constructor(driver: Driver) {
+    this.driver = driver;
+  }
+
+  async check_pageIsLoaded(): Promise<void> {
```

Comment:
Could you clarify the use of the underscore separator and the `check_` prefix? Are we formalizing it as a pattern? If so, would it make sense to document this for contributors?

Reply:
yes we formalize it as a pattern and have the naming convention in the contributor documentation.https://github.com/MetaMask/contributor-docs/blob/main/docs/testing/e2e/extension-e2e-guidelines.md#check_-methods